server/controllers/api.py

```diff
@@ -923,6 +932,19 @@ def post(self, user, backup_id):
 
         return self.schema.store_comment(user, backup)
 
+    @marshal_with(schema.get_fields)
+    def get(self, user, backup_id):
+        backup = models.Backup.query.get(backup_id)
+        if not backup:
+            if user.is_admin:
+                restful.abort(404)
+            else:
+                restful.abort(403)
+        if not models.Backup.can(backup, user, "view"):
+            restful.abort(403)
+        return {"comments": [backup.comments]}
```

Comment:
This won't actually accomplish what you want. The return statement is checked against the schema and only the fields that are in the schema are returned. In this case none of the fields in the return match the schema - so the api won't respond with anything meaningful. 

If you want to the schema to be a list of comments - take a look at the VersionSchema and how it accomplishes that. The get_fields is a list of another schema. 

This is pretty close though.


You can manually test this by going to a backup in your browser once you are running the server (and have setup the testing database) 

> http://localhost:5000/api/v3/backups/<backup id>/comment/